Maintenance.md

```diff
@@ -1,6 +1,64 @@
 (Version >= 1.19.0)
 
+# Description
+
 This feature is designed for:
 
 - Temporary disable notifications during a time window
 - Show a maintenance message on status pages.
+
+# Usage
+
+You can access the maintenance management via the menu at the top left.
+
+![Docker host monitor](img/maintenance/user-menue-Maintenace.png)
+
+Within the maintenance management, there is an overview of the currently configured maintenance windows.
+In the upper area, you can configure another maintenance using the '+ Schedule Maintenance' button.
+
+![Docker host monitor](img/maintenance/schedule-maintenance-button.png)
+
+## New Schedule Maintenance
```

Comment:
I don't believe this paragraph needs documentation.
The reason I would like to not document every single field is that this gets stale quickly.

Is there some context/.. I am missing?
Was one of the fields of the form in our frontend unclear i what it does?

Reply:
I think we need more information in the documentation for the maintenance feature, because this page is linked from the bottom of the maintenance page titled "[Learn More](https://github.com/louislam/uptime-kuma/wiki/Maintenance)", but there is currently no way to learn more about the maintenance feature.

Whether an input field is unclear or not belongs to the inexperienced user to ask and if in doubt, simply explain briefly.

We should at least explain what effect a maintenance entry has and how the selected monitors in the system behave during the period.